MoltenVK/MoltenVK/Commands/MVKCommandBuffer.mm

```diff
@@ -551,17 +578,34 @@
     _occlusionQueryState.endMetalRenderPass();
 }
 
+void MVKCommandEncoder::recordTimestamp(id<MTLCommandEncoder> commandEncoder)
```

Comment:
This parameter declaration results in a build warning because `MTLCommandEncoder` doesn't actually define `sampleCountersInBuffer:atSampleIndex:withBarrier:`, I guess because the `MTLParallelRenderCommandEncoder` subclass doesn't support it.

Replacing `id<MTLCommandEncoder> commandEncoder` with `id commandEncoder` removes the build warning, and is no less safe, since you already check for `respondsToSelector`.